### PR TITLE
Login form and progress bar edits

### PR DESCRIPTION
**Pull request by samjsh**
Please note: No edits made to PageContent that affect functionality. Upon saving the page my IDE made some spacing changes and changed ' to "" and I just left it.

{{CHANGED FUNCTIONALITY}}
Edited the ProgressBar code to make it custom for this project and implemented it into the ProgramDetailPage & the SessionDetailPage. Have also added 'PageContent' to ProgramDetailPage to make it more consistent.

### CODE REVIEW

**Review by Simplicity305 — CHANGES_REQUESTED** [review]
Hey, could you please remove the image from the login form?

Also, did you make any changes at that level? The PR and branch name indicates login form but cant see any changes for this?

File: src/Components/ProgressBar/ProgressBar.jsx [hunk 1]
```diff
@@ -1 +1,34 @@
-// (SAM) - TO DO
\ No newline at end of file
+const ProgressBar = (props) => {
+  const { completed } = props;
+
+  const containerStyles = {
```
Comment (builtbyproxy) [inline src/Components/ProgressBar/ProgressBar.jsx:4]: [This is a good read](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx#:~:text=Naming%20Convention%20for%20Constants) for the naming convention of these `const` variables 👍

File: src/Pages/ProgramDetailPage.jsx [hunk 2]
```diff
@@ -2,40 +2,55 @@
...
+            <ToggleButton />
+          </div>
+          <ProgressBar
+            completed={
+              programData.mentors_required > 0
+                ? (programData.mentors_assigned /
+                    programData.mentors_required) *
+                  100
+                : 0
+            }
```
Comment (builtbyproxy) [inline src/Pages/ProgramDetailPage.jsx:47]: I'd try and write the calculation itself a little more readable on one line like so: 
```suggestion
            completed={
              programData.mentors_required > 0
                ? (programData.mentors_assigned / programData.mentors_required) * 100
                : 0
            }
```

The indentation of `?` and `:` on their own lines is really nice for readability! Great work 🎉

**Comment by samjsh** [comment]
> Hey, could you please remove the image/logo from the login form?
> 
> Also, did you make any changes at that level? The PR and branch name indicates login form but cant see any changes for this?

Hey, yep initially I intended to make edits to the LoginForm and the ProgressBar in the same work but ended up just working on the ProgressBar. Do we still need to the LoginForm edits before merging this as I plan to attack this in another piece of work. I just want to get the progress bar in before any conflicts.

**Pull request merged**